docs/migration-to-NRI.md

```diff
@@ -0,0 +1,90 @@
+# Enabling CRI-RM
+
+## 1. Install the package
+
+For Fedora, CentOS, SUSE:
+ - CRIRM_VERSION=`curl -s "https://api.github.com/repos/intel/cri-resource-manager/releases/latest" | \
+               jq .tag_name | tr -d '"v'`
+ - source /etc/os-release
+ - [ "$ID" = "sles" -o "$ID" = "opensuse-leap" ] && export ID=suse
+ - sudo rpm -Uvh https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/cri-resource-manager-${CRIRM_VERSION}-0.${ID}-${VERSION_ID}.x86_64.rpm
+
+For Ubuntu, Debian:
+ - CRIRM_VERSION=`curl -s "https://api.github.com/repos/intel/cri-resource-manager/releases/latest" | \
+               jq .tag_name | tr -d '"v'`
+ - source /etc/os-release
+ - pkg=cri-resource-manager_${CRIRM_VERSION}_${ID}-${VERSION_ID}_amd64.deb; curl -LO https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/${pkg}; sudo dpkg -i ${pkg}; rm ${pkg}
+
+## 2. Setup & verify:
+ - sudo cp /etc/cri-resmgr/fallback.cfg.sample /etc/cri-resmgr/fallback.cfg
+ - sudo systemctl enable cri-resource-manager && sudo systemctl start cri-resource-manager
+ - systemctl status cri-resource-manager
+
+## 3. Run: "systemctl cat kubelet". Look for: EnvironmentFile=/.../kubeadm-flags.env
+
+## 4. Run: "sudo vim /.../kubeadm-flags.env"
+
+## 5. Change the --container-runtime-endpoint from containerd socket to cri-resmgr.scok:
+ - KUBELET_KUBEADM_ARGS="--container-runtime-endpoint=unix:///var/run/cri-resmgr/cri-resmgr.sock --pod-infra-container-image=registry.k8s.io/pause:3.9"
+
+ Alternatively run:
+ - kubelet <other-kubelet-options> --container-runtime=remote \
+     --container-runtime-endpoint=unix:///var/run/cri-resmgr/cri-resmgr.sock
+
+## 6. Restart kubelet: "systemctl restart kubelet"
+
+## 7. Run CRI-RM with your desired policy with the following command:
+ - cri-resmgr --force-config <config-file> --runtime-socket unix:///var/run/containerd/containerd.sock
+
+## 8. Deploy your pod
+
+## 9. See what resources the container got assigned with:
+ - kubectl exec -c $container $pod -- cat /proc/self/status | grep _allowed_list
+
+Output should be something along the lines of:
+
+Cpus_allowed_list:	10-15 </br>
+Mems_allowed_list:	1
+
```

Comment:
Generally this looks good, but I think we need to adjust and reorganize the content a bit and fill in a few still missing details. See my more detailed train of thoughts below... 

Considering the audience, I wonder if the first section ('Enabling CRI-RM') is really necessary. I think the target audience for this documentation is folks who are already up and running with CRI Resource Manager and would like/need to switch to using one the NRI-retrofitted version of their policy.

For such folks, most of the necessary information to make the switch is already present in the subsequent chapters, but I think it would be good to perhaps structure it slightly differently and add a few missing details/bits. So maybe we could restructure the outline of the description like this:

1. Prerequisites
  -  up and running with CRI Resource Manager
  - one of two supported policies in use: balloons or topology-aware
  - anything else needs to be 'ported', a mechanical task following the example of either balloons or topology-aware
2. Steps for an initial/basic migration test
  - this is basically what you have already described, basically the steps outlined below
  - clone repo, build policies and images
  - enable NRI in runtime (containerd, and I also would describe how to enable it for CRI-O because of the following chapter)
  - import policy image
  - give it a test run, check that things work fine, policy gets applied to sample workloads
3. Steps for a more real-life migration, assuming folks run a self-hosted repository where they can publish images
  - same as the steps for above, but describe what needs to be done to avoid having to manually import images
  - build images
  - push images to self-hosted repo
  - change deployment file to reference image in self-hosted repo
4. Migrating existing configuration
  - the ConfigMap used by the ported policies/infra has a different name/naming scheme than the original one used in CRI-RM, so let's point/spell out the differences
5. Migrating existing workloads
  - the annotation one can use to customize how a policy treats a workload use slightly different keys than the original ones in CRI-RM, so let's point/spell out the differences